### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by jetfuel**
https://github.com/PaddlePaddle/PaddlePaddle.org/issues/283
https://github.com/PaddlePaddle/PaddlePaddle.org/issues/284

### CODE REVIEW

File: portal/locale/zh/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -126,11 +126,11 @@ msgid ""
 "\"_blank\">install Docker</a> on your computer. After installing Docker, "
 "create a new file called "
 msgstr ""
-"之後 <a href=\"https://docs.docker.com/engine/installation/\" target=\"_blank"
+"<a href=\"https://docs.docker.com/engine/installation/\" target=\"_blank"
 "\">安装 Docker</a> 在你的电脑上。安装后，创建 "
```
Comment (pengli09) [inline portal/locale/zh/LC_MESSAGES/django.po]: 在你的电脑上<a href=\"https://docs.docker.com/engine/installation/\" target=\"_blank\">安装 Docker</a>。

Comment (luotao1) [reply portal/locale/zh/LC_MESSAGES/django.po]: 你-》您

**Comment by jetfuel** [comment]
Also https://github.com/PaddlePaddle/PaddlePaddle.org/issues/274

File: portal/locale/zh/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -122,16 +122,17 @@ msgstr "和在电脑上安装PaddlePaddle"
...
+"在您的电脑上<a href=\"https://docs.docker.com/engine/installation/\" target="
+"\"_blank\">安装Docker。</a>"
 
 #: portal/templates/index.html:112
-msgid " in an new and empty directory:"
-msgstr " 在一个新的空目录中："
+msgid ""
+"After installing Docker, create a new file called <em>housing.py</em> in a "
+"new and empty directory:"
+msgstr "安裝好後，在一個新的目錄下创建一个<em>housing.py</em> "
```
Comment (pengli09) [inline portal/locale/zh/LC_MESSAGES/django.po]: 请注意用使用简体中文
安装好后，在一个新的目录下创建一个

Comment (jetfuel) [reply portal/locale/zh/LC_MESSAGES/django.po]: Good catch. Already changed in the code. Will wait for next deployment.

**Pull request merged**